esphome/components/esp32_camera_web_server/web_server.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#ifdef ARDUINO_ARCH_ESP32
+
+#include "esphome/core/component.h"
+#include "esphome/core/helpers.h"
+#include "esphome/core/preferences.h"
+#include "esphome/components/esp32_camera/esp32_camera.h"
+
+struct httpd_req;
+
+namespace esphome {
+namespace esp32_camera_web_server {
+
+enum Mode { Stream, Snapshot };
+
+class WebServer : public Component {
```

Comment:
```suggestion
class CameraWebServer : public Component {
```

Would be better to prevent confusion with the `web_server` component.